src/ui/digital-exchange/components/list/ComponentListContent.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Row } from 'patternfly-react';
+
+import ComponentListContainer from 'ui/digital-exchange/components/list/ComponentListContainer';
+
+const ComponentListContent = () => (
```

Comment:
it sounds a bit confusing to me: why a `ComponentListContent` contains a `ComponentListContainer`?

Reply:
@gidesan hierarchy here looks like this: 

`ComponentListContent ` > `ComponentListContainer` > `ComponentList`

I understand your concern but `ComponentListContent` is considered as more broad component, which can encapsulate everything(content) related to component list while `ComponentListContainer` is direct parent of `ComponentList` and also connects component with redux store. 

This kind of hierarchy and logic is applied to  other parts of the project, so I decided to follow that. You can check `FragmentListContent` for an example. 

While hierarchy seems correct to me I think we can improve the `ComponentListContent` name to be more intuitive.

Reply:
But in your example a `FragmentListContent ` contains `FragmentSearchFormContainer`, not a `FragmentListContainer`. And this could make sense, because are content and container related to different components.

Reply:
@gidesan it also contains `FragmentListTableContainer` which is same level as `ComponentListContainer`, it just has Table in the name to indicate that it is a table.

Reply:
Ok guys, you are both right in this instance: you may end up with a component that is containing several other components, which might be containers used to connect the child components to the redux store, just as @otarza pointed out. In this instance though it seems that the only purpose of this component is currently literally containing one container, which kinda looks like a poltergeist anti-pattern. If we are literally one PR away from adding additional containers/components in the body of this one we can leave it there, otherwise this is going to be an unnecessary wrapper to future proof something that might never exist.

please act accordingly

Reply:
Thanks @nicpuddu, 

@gidesan let's remove for now and we will add it in the future if we need it.